package.json

```diff
@@ -28,7 +28,7 @@
     "html-webpack-plugin": "^3.2.0",
     "mini-css-extract-plugin": "^0.4.0",
     "prop-types": "^15.6.1",
-    "react": "^16.4.0",
+    "react": "^16.4.1",
```

Comment:
No reason to change this, but I was trying to see if the version was the source of the error in other projects.